### ISSUE TITLE
summary.survfit does not subset std.chaz as it does chaz

### ISSUE THREAD

**Issue by gmbecker** [therneau/survival#122]
It appears that the std.chaz component is not subset, but the chaz component **is** when summarizing a `survfit` object.

If I am understanding the redirection/code-generation correctly, I think adding "std.cumhaz" to line 430 here:
https://github.com/therneau/survival/blob/787027105bcc3d06b2eac00d360bef65f5dce508/noweb/survfitms.Rnw#L430-L432
would resolve this.

The reason I care about this is that a) I thought you might want to know, but b) I want to compute on `summary.survfit` objects and this throws a (minor) wrench in their translation to a data.frame.

Illustration of the issue "in action".
```
>  fit <- survfit(Surv(time, status) ~ x, data = aml) 
> str(fit)
List of 17
 $ n        : int [1:2] 11 12
 $ time     : num [1:20] 9 13 18 23 28 31 34 45 48 161 ...
 $ n.risk   : num [1:20] 11 10 8 7 6 5 4 3 2 1 ...
 $ n.event  : num [1:20] 1 1 1 1 0 1 1 0 1 0 ...
 $ n.censor : num [1:20] 0 1 0 0 1 0 0 1 0 1 ...
 $ surv     : num [1:20] 0.909 0.818 0.716 0.614 0.614 ...
 $ std.err  : num [1:20] 0.0953 0.1421 0.1951 0.2487 0.2487 ...
 $ cumhaz   : num [1:20] 0.0909 0.1909 0.3159 0.4588 0.4588 ...
 $ std.chaz : num [1:20] 0.0909 0.1351 0.1841 0.233 0.233 ...
 $ strata   : Named int [1:2] 10 10
  ..- attr(*, "names")= chr [1:2] "x=Maintained" "x=Nonmaintained"
 $ type     : chr "right"
 $ logse    : logi TRUE
 $ conf.int : num 0.95
 $ conf.type: chr "log"
 $ lower    : num [1:20] 0.754 0.619 0.488 0.377 0.377 ...
 $ upper    : num [1:20] 1 1 1 0.999 0.999 ...
 $ call     : language survfit(formula = Surv(time, status) ~ x, data = aml)
 - attr(*, "class")= chr "survfit"
> fitsum <- summary(fit, times = c(20, 30))
> str(fitsum)
List of 19
 $ n            : int [1:2] 11 12
 $ time         : num [1:4] 20 30 20 30
 $ n.risk       : num [1:4] 7 5 6 4
 $ n.event      : num [1:4] 3 1 5 3
 $ n.censor     : num [1:4] 1 1 1 0
 $ surv         : num [1:4] 0.716 0.614 0.583 0.292
 $ std.err      : num [1:4] 0.14 0.153 0.142 0.139
 $ cumhaz       : num [1:4] 0.316 0.459 0.492 1.108
 ```
**`$ std.chaz     : num [1:22] 0 0.0909 0.1351 0.1841 0.233 ...`**
```
 $ strata       : Factor w/ 2 levels "x=Maintained",..: 1 1 2 2
 $ type         : chr "right"
 $ logse        : logi TRUE
 $ conf.int     : num 0.95
 $ conf.type    : chr "log"
 $ lower        : num [1:4] 0.488 0.377 0.362 0.115
 $ upper        : num [1:4] 1 0.999 0.941 0.741
 $ call         : language survfit(formula = Surv(time, status) ~ x, data = aml)
 $ table        : num [1:2, 1:9] 11 12 11 12 11 ...
  ..- attr(*, "dimnames")=List of 2
  .. ..$ : chr [1:2] "x=Maintained" "x=Nonmaintained"
  .. ..$ : chr [1:9] "records" "n.max" "n.start" "events" ...
 $ rmean.endtime: num [1:2] 103 103
 - attr(*, "class")= chr "summary.survfit"
> sessionInfo()
R version 4.0.2 (2020-06-22)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Mojave 10.14.6

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRblas.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] dplyr_1.0.2      broom_0.7.0      rtables_0.3.2.11 magrittr_1.5    
[5] survival_3.1-12 

loaded via a namespace (and not attached):
 [1] splines_4.0.2    tidyselect_1.1.0 lattice_0.20-41  R6_2.4.1        
 [5] rlang_0.4.7      fansi_0.4.1      tcltk_4.0.2      tools_4.0.2     
 [9] grid_4.0.2       utf8_1.1.4       cli_2.0.2        htmltools_0.5.0 
[13] ellipsis_0.3.1   digest_0.6.25    assertthat_0.2.1 tibble_3.0.3    
[17] lifecycle_0.2.0  crayon_1.3.4     Matrix_1.2-18    purrr_0.3.4     
[21] tidyr_1.1.1      vctrs_0.3.2      glue_1.4.1       compiler_4.0.2  
[25] pillar_1.4.6     generics_0.0.2   backports_1.1.9  pkgconfig_2.0.3 
```

**Comment by therneau** [comment]
You are exactly right, and I've fixed this in my master copy.  Once I chase down another issue, and pass R CMD check, I'll update github.   (My master copy has been on sccs  then rcs then svn then mercurial over the decades.  I still prefer mercurial to git.  And updating git only after I test has kept me from inflicting multiple stupid mistakes on all of you.)

**Comment by therneau** [comment]
Now repaired
